src/themes/colors.ts

```diff
@@ -0,0 +1,89 @@
+import type { ThemeItem } from '@types'
+
+type BackgroundColorKeys =
+  | 'grey01'
+  | 'grey02'
+  | 'grey03'
+  | 'grey04'
+  | 'primary'
+  | 'primaryWeak'
+  | 'white'
+type BackgroundColorValues =
+  | '#FAFAFA'
+  | '#F6F6F7'
+  | '#F3F3F3'
+  | '#EDEEEF'
+  | '#FF6E59'
+  | '#FFF0EE'
+  | '#FFFFFF'
+type BrandColorKeys = 'primary' | 'primaryWeak' | 'sub' | 'subWeak'
+type BrandColorValues = '#FF6E59' | '#FFECE9' | '#673CE6' | '#E1D8FA'
+type ActionColorKeys = 'success' | 'error'
+type ActionColorValues = '#4AB783' | '#F36140'
+type GreyScaleColorKeys =
+  | 'white'
+  | 'grey05'
+  | 'grey10'
+  | 'grey20'
+  | 'grey30'
+  | 'grey50'
+  | 'grey70'
+  | 'grey90'
+  | 'black'
+type GreyScaleColorValues =
+  | '#FFFFFF'
+  | '#F6F6F7'
+  | '#E8E8EA'
+  | '#D1D3D6'
+  | '#C6C8CC'
+  | '#929399'
+  | '#65646A'
+  | '#2F2E36'
+  | '#000000'
+type DimColorKeys = 'opacity50' | 'opacity70'
+type DimColorValues = '#00000080' | '#000000B2'
+
+export interface Colors {
+  background: ThemeItem<BackgroundColorKeys, BackgroundColorValues>
+  brand: ThemeItem<BrandColorKeys, BrandColorValues>
+  action: ThemeItem<ActionColorKeys, ActionColorValues>
+  greyScale: ThemeItem<GreyScaleColorKeys, GreyScaleColorValues>
+  dim: ThemeItem<DimColorKeys, DimColorValues>
+}
+
+export const colors: Colors = {
+  action: {
+    error: '#F36140',
+    success: '#4AB783'
+  },
+  background: {
+    grey01: '#FAFAFA',
+    grey02: '#F6F6F7',
+    grey03: '#F3F3F3',
+    grey04: '#EDEEEF',
+    primary: '#FF6E59',
+    primaryWeak: '#FFF0EE',
+    white: '#FFFFFF'
+  },
+  brand: {
+    primary: '#FF6E59',
+    primaryWeak: '#FFECE9',
+    sub: '#673CE6',
+    subWeak: '#E1D8FA'
+  },
+  dim: {
+    opacity50: '#00000080',
+    opacity70: '#000000B2'
+  },
+  greyScale: {
+    black: '#000000',
+    grey05: '#F6F6F7',
+    grey10: '#E8E8EA',
+    grey20: '#D1D3D6',
+    grey30: '#C6C8CC',
+    grey50: '#929399',
+    grey70: '#65646A',
+    grey90: '#2F2E36',
+    white: '#FFFFFF'
+  }
+}
```

Comment:
```suggestion
} as const

type BackgroundColorKeys = keyof typeof colors.background
type BackgroundColorValues = typeof BackgroundColors[BackgroundColorKey]
```

객체를 위한 타입이 필요한 상황이라면 이렇게 사용해보는게 어떨까용 !?
1. 사용하는 쪽에서 어떤 type이 있는지 더욱 명확하게 보임
2. 간결해진 타입 작성
3. 추후 옵션이 추가될 경우, type, object 두  번 수정 대신 object만 수정

**AS-IS**
<img width="775" alt="image" src="https://user-images.githubusercontent.com/47546413/185415692-80b2ba83-79f0-481d-8eea-059678bc855d.png">

**TO-BE**
<img width="967" alt="image" src="https://user-images.githubusercontent.com/47546413/185413123-f19c5a8c-e457-46ed-be88-01e6b96faac8.png">

Reply:
오 좋은 것 같아 type 선언 후에 object 지정할 때 편하니까 type => object만 생각했던 것 같아
객체에서 키값을 뽑아오는게 더 나은 방법같아 이고 다른 파일들도 같이 수정할게 👍